lib/hex_web/email/sender.ex

```diff
@@ -0,0 +1,24 @@
+defmodule HexWeb.Email.Sender do
```

Comment:
let's leave `HexWeb.Mail` and `HexWeb.Email` as is, as they serve different purposes (the latte r is more like UserEmail or EmailAddress really.)

I'd be ok with renaming `HexWeb.Mail` to `HexWeb.Emails` but let's leave it for some other time (btw: https://github.com/thoughtbot/bamboo/issues/238)